### PR TITLE
Feature/profiles

### PR DESCRIPTION
**Pull request by GPayne**
Feature for saving current settings to a new profile.

### CODE REVIEW

File: data/assets/default.profile [hunk 1]
```diff
@@ -3,10 +3,10 @@
 
 #Asset
 scene/solarsystem/planets/earth/earth	required
-scene/solarsystem/planets/earth/satellites/satellites   required
+scene/solarsystem/planets/earth/satellites/satellites	required
 
 #Property
-setPropertyValue  {earth_satellites}.Renderable.Enabled false
+setPropertyValue	{earth_satellites}.Renderable.Enabled	false
```
Comment (GPayne) [inline data/assets/default.profile:9]: This is a tab-delimited file. If these are replaced with spaces, then the parsing won't work.

File: openspace.cfg [hunk 2]
```diff
@@ -64,7 +64,7 @@ Asset = "default"
 -- Sets the profile that should be loaded by OpenSpace.  Profiles are going to replace
 -- assets in a future versions and shouldn't be used at the same time as the 'Asset'
 -- setting above
--- Profile = "default"
+Profile = "default"
```
Comment (GPayne) [inline openspace.cfg:67]: This will make OpenSpace master run in profile mode by default!

File: data/assets/default.profile (hunk 1 above)
Comment (alexanderbock) [reply data/assets/default.profile:9]: They were space-delimited in the repository and I replaced them with tabs on my end

File: openspace.cfg (hunk 2 above)
Comment (alexanderbock) [reply openspace.cfg:67]: Yea, I left there for now as I had to quickly commit everything when I left for the office this morning.  Discussion pending if we want to make it the default

**Comment by alexanderbock** [comment]
Punting the review back to you @GPayne

**Pull request merged**